src/widgetastic_patternfly/__init__.py

```diff
@@ -1537,6 +1537,9 @@ def _verify_enabled(self):
         if not self.is_enabled:
             raise DropdownDisabled('Dropdown "{}" is not enabled'.format(self.text))
 
+    def read(self):
+        return self.browser.text(self.BUTTON_LOCATOR, parent=self)
```

Comment:
I think this is a bit backwards - `read` should call `currently_selected`, which should be defined here in Dropdown.

Both methods should be in Dropdown.